app/v1/models/response/retrieveBusinessDetails/des/BusinessDetails.scala

```diff
@@ -53,11 +54,24 @@ case class BusinessDetails(businessId: String,
 }
 
 object BusinessDetails {
-  implicit val reads: Reads[BusinessDetails] = (
+
+  val cashOrAccrualsReads: Reads[AccountingType] = (JsPath \ "cashOrAccrualsFlag").read[Boolean].map {
+    case false => CashOrAccruals.`cash`
+    case true => CashOrAccruals.`accruals`
+  }.map(_.toMtd)
+
+  implicit val accountingPeriodReads: Reads[Seq[AccountingPeriod]] = (
```

Comment:
It's not really a big thing, but this doesn't need to be implicit as you're only using it explicitly